JSDemos/Demos/Charts/PolarChartAnnotations/description.md

```diff
@@ -1,12 +1,12 @@
 Annotations are images and text blocks that provide additional information about the visualized data.
 
-To configure annotations, assign an array of objects to the [annotations](https://js.devexpress.com/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](https://js.devexpress.com/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
+To configure annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
 
-Each annotation's [type](https://js.devexpress.com/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/annotations/#type) attribute should be set to *"text"* or *"image"*. In this demo, all annotations have the *"text"* type.
+Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/annotations/#type) attribute should be set to *"text"* or *"image"*. In this demo, all annotations have the *"text"* type.
```

Comment:
The word "should" leaves some uncertainty. I "should" implies that maybe I don't do it. What happens then? If there is no way this property can be set to anything other than "text" or "image", I wouldn't use the word "should". 

 ```suggestion
You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/annotations/#type) attribute to *"text"* or *"image"*. This demo shows only *"text"* annotations.
```